src/components/CardComponent.jsx

```diff
@@ -1,27 +1,34 @@
-import React, { Component } from 'react'
-import { Card } from 'semantic-ui-react'
-
-class CardComponent extends React.Component {
-    statistics = this.props.statistics;
-
-    render() {
-        let cards = this.statistics.map((value, index) => {
-            return (
-                <Card key={index}>
-                    <Card.Content>
-                        <Card.Header>{value.value}</Card.Header>
-                        <Card.Description>{value.label}</Card.Description>
-                    </Card.Content>
-                </Card>
-            )
-        });
-
-        return (
-            <Card.Group centered>
-                {cards}
-            </Card.Group>
-        )
-    }
-}
-
-export default CardComponent;
+import React, { Component } from 'react';
+import { Card } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+class CardComponent extends Component {
+    statistics = this.props.statistics;

```

Comment:
The starting point of `statistics` and `render` should be 2 spaces from `class`, not 4 spaces. Please move back by 2 spaces